### PR TITLE
[FIX] stock: manage mrp_operation picking type

### PR DESCRIPTION
**Pull request by GaetanVandenBergh**
mrp_operation picking type is not handle by default in stock module get_description function. 

Add default return to stock get_description function

opw-3232811

Description of the issue/feature this PR addresses:
Handle "mrp_operation" picking type

Current behavior before PR:
Unable to confirm MO with "mrp_operation" picking type for stock move

Desired behavior after PR is merged:
Able to confirm MO with "mrp_operation" picking type for stock move

---
I confirm I have signed the CLA and read the PR guidelines at www.odoo.com/submit-pr

### CODE REVIEW

File: addons/stock/models/product.py [hunk 1]
```diff
@@ -195,19 +195,23 @@ def get_components(self):
...
         picking_code = picking_type_id.code
-        description = self.description or self.name
+        description = self._get_description_or_name()
         if picking_code == 'incoming':
             return self.description_pickingin or description
         if picking_code == 'outgoing':
             return self.description_pickingout or self.name
         if picking_code == 'internal':
             return self.description_picking or description
+        return description
```
Comment (clesgow) [inline addons/stock/models/product.py:214]: Pretty sure you can achieve the same results with only this single line :wink:

**Review by clesgow — CHANGES_REQUESTED** [review]
Thanks for the PR!
A little something about the commit message :
- Since your commit is related to a ticket, it should be flagged as `opw-xxxxxx` instead of `task-xxxxxx`
- Try to stick to the default git commit's message length for each line (which should be 72 characters for the message body). Here the lines are too long.

Due to the comment, I don't think #115309 is even necessary in this case.

File: addons/stock/models/product.py (hunk 1 above)
Comment (GaetanVandenBergh) [reply addons/stock/models/product.py:214]: Yes, but as mrp_operation is added via mrp module, I though it's was good thing to handle it "properly" in mrp and to set a fallback in stock.
But of course, I can just keep this line if it's better

Comment (GaetanVandenBergh) [reply addons/stock/models/product.py:214]: Not sure to understand why https://github.com/odoo/odoo/pull/115309 is not necessary, it's the not really the same fix, as 15.0 manage HTML description

Comment (GaetanVandenBergh) [reply addons/stock/models/product.py:211]: Ok yes indeed, without the overriding way, it's the same fix

Commit and PR updated to reflect the review, other PR closed
Thanks

**Comment by ticodoo** [comment]
@robodoo r+

**Pull request closed without merging**